dietpi/dietpi-software

```diff
@@ -5293,6 +5303,27 @@ Package: wireguard wireguard-dkms wireguard-tools\nPin: release n=bullseye\nPin-
 
 		fi
 
+		software_id=177 # Firefox Sync Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# Dependencies
+			DEPS_LIST='python-virtualenv libffi-dev libssl-dev'
+			# - Pre-compiling required on ARM
+			(( $G_HW_ARCH < 10 )) && DEPS_LIST+=' libffi-dev libssl-dev'
+
+			# Download & Install
+			INSTALL_URL_ADDRESS='https://github.com/mozilla-services/syncserver/archive/master.tar.gz'
+			Download_Install "$INSTALL_URL_ADDRESS"
+			mv syncserver /opt/firefox-sync
```

Comment:
Since `/opt/firefox-sync` does not contain settings/database anymore, we can remove it for reinstalls, if existing, else `mv` would fail:
```
[[ -d '/opt/firefox-sync' ]] && rm -R /opt/firefox-sync
```